translations/dicts.go

```diff
@@ -1,9 +1,9 @@
-// Code generated by go-bindata. (@generated) DO NOT EDIT.
-
- //Package translations generated by go-bindata.// sources:
+// Code generated for package translations by go-bindata DO NOT EDIT. (@generated)
+// sources:
```

Comment:
This file in general has a lot of spurious diffs, which I tend to avoid in my projects for multiple reasons. @xxxserxxx do you mind those?

I use github.com/go-bindata/go-bindata 3.1.3 and I dont have this problem. (to be fair there are multiple forks of this tool it's annoying) Which one did you use?

If you want a patch see 592228881e67150617e0223af7680062ba4e8797

Reply:
I prefer to minimize diffs for probably the same reasons, but I'm not going to force people to correct them. Thanks for doing that, @lourkeur, and thank you @donPatino for the translation. I'll try to find time to get it reviewed and merged this weekend.

I have a version ready to go out that eliminates go-bindata (given the recently added official Go embed functionality); I just haven't had time to put a release together.  That should reduce the amount of spurious diffs.

Reply:
I used the install instructions from github.com/go-bindata/go-bindata & it seems to have installed 3.1.2 by default; I had to download the source and build 3.1.3. I now have 3.1.3 on my dev machine, how should I go about correcting this?
- Create a new commit using go-bindata 3.1.2?
- Delete my current commit and commit again with go-bindata 3.1.3?

Reply:
I followed the install instructions in github.com/go-bindata/go-bindata & it seems to have installed 3.1.2 by default. I have now built 3.1.3 on my development machine; how should I go about correcting this?
- Create a new commit using go-bindata 3.1.3
- Remove current commit & create a new one using go-bindata 3.13

Reply:
option number two, replacing the commit, (or equivalently the `--amend` commit option) is preferred.

Reply:
Great, will amend.
Using version 3.1.3 resolved most of the spurious diffs; except the changes to the FileMode are still generated as "436" rather than "420". Any suggestions as to what is causing this?

Example:
```
info := bindataFileInfo{name: "de_DE.toml", size: 7223, mode: os.FileMode(436), modTime: time.Unix(1625599
921, 0)}
```

go-bindata versions:
```
$ go-bindata --version
go-bindata 3.1.3 (Go runtime go1.14).
Copyright (c) 2010-2013, Jim Teeuwen.
```

Command I am running (from the root of the project):
```
$ go generate
```

Please let me know if there is additional information that could help with this.

Reply:
It's unix permissions on the files. 420 = 0644, 436 = 0664. That's a trivial matter, don't worry too much about permanently fixing it on your end. We can handle this manually.